int16 centerPointX
int16 centerPointY
int16 size
